src/tests/config/mailConfig.test.js

```diff
@@ -1,12 +1,14 @@
 import chai from 'chai';
 import mockery from 'mockery';
+import dotenv from 'dotenv';
 import nodemailerMock from 'nodemailer-mock';
 import sendMailer from '../../config/mailConfig';
 
 const { expect } = chai;
+dotenv.config();
 
 let info, error;
-const receivers = ['nedyudombat@gmail.com'];
+const receivers = [`${process.env.TEST_MAIL}`];
```

Comment:
Since this is just for testing we don't need to put the email in an environmental variable.